cargo/cargo_build_script_runner/bin.rs

```diff
@@ -57,13 +57,24 @@ fn main() {
                 .current_dir(manifest_dir.clone())
                 .env("OUT_DIR", out_dir_abs)
                 .env("CARGO_MANIFEST_DIR", manifest_dir)
-                .env("RUSTC", rustc);
+                .env("RUSTC", rustc)
+                .env("RUST_BACKTRACE", "full");
 
             if let Some(cc) = cc {
                 command.env("CC", cc);
             }
 
-            let output = BuildScriptOutput::from_command(&mut command);
+            let output = BuildScriptOutput::from_command(&mut command).unwrap_or_else(|exit_code| {
+                eprintln!(
+                    "Build script process failed{}",
```

Comment:
opt: Can we make this error stands out in some way? When I was debugging this code it was kind of hard to figure out if the error was coming from the build script or from the runner, that would be nice to make this really stands out (also to see it from the bazel output).